tools/scripts/used-headers.txt

```diff
@@ -39,6 +39,7 @@ cstdlib
 cstring
 ctime
 fstream
+functional
```

Comment:
@elBoberido Wouldn't it be possible to use `cxx::function` or maybe even `cxx::function_ref`?

Reply:
I have to look why exactly this is used. I just copied my proof of concept to check if it builds and will now remove everything that is not needed

Reply:
@mossmaurice removed